Cython/Compiler/ExprNodes.py

```diff
@@ -9632,10 +9632,18 @@ class GeneratorExpressionNode(LambdaNode):
     #
     # loop      ForStatNode   the for-loop, containing a YieldExprNode
     # def_node  DefNode       the underlying generator 'def' node
+    # call_parameters [ExprNode]   (Internal) parameters passed to the DefNode call
 
     name = StringEncoding.EncodedString('genexpr')
     binding = False
 
+    child_attrs = LambdaNode.child_attrs + ["call_parameters"]
+    subexprs = LambdaNode.subexprs + ["call_parameters"]
```

Comment:
`LambdaNode` (which `GeneratorExpressionNode` inherits from) specifically sets `child_attrs` and so the link between them is broken. I didn't want to change it in `LambdaNode` since I  wasn't sure if it was deliberate so I did both here.

Reply:
Interesting. Maybe `LambdaNode` just does it wrong and should extend `subexprs` instead? Could even be the reason for #3267.

Reply:
It looks like `LambdaNode.def_node` wants to be in `child_attrs` but not `subexprs` because it isn't an expression so doesn't have `generate_evaluation_code` so that bit is deliberate. Presumably that does stop some `LambdaNode`'s inherited `subexprs` from being processed, but explicitly including them in `child_attrs` doesn't seem to fix #3267.